crates/enarx-config/src/lib.rs

```diff
@@ -77,15 +77,25 @@ fn default_addr() -> String {
 /// See the [crate] documentation for examples.
 pub struct FileName(String);
 
-impl From<String> for FileName {
-    fn from(value: String) -> Self {
-        Self(value)
+impl TryFrom<String> for FileName {
+    type Error = String;
+
+    fn try_from(name: String) -> Result<Self, Self::Error> {
+        for b in name.chars() {
+            match b {
+                c @ ':' => return Err(format!("file name must not contain '{c}'")),
```

Comment:
maybe add "/" as well to satisfy clippy

Reply:
Giving it another thought, `:` should actually be a valid file name on the FS and `FD_NAMES` will go away, since we'll always have 4 FDs open:
- stdin (null by default)
- stdout (null by default)
- stderr (null by default)
- VFS root

I'll just replace `:` by `/` once I do the change.